compiler/src/test/java/com/squareup/anvil/compiler/dagger/MembersInjectorGeneratorTest.kt

```diff
@@ -1627,6 +1627,276 @@ public final class InjectClass_MembersInjector<T, U, V> implements MembersInject
     }
   }
 
+  @Test
+  fun `a member injector is generated for a class with two super classes in another module`() {
+
+    val otherModuleResult = compile(
+      """
+      package com.squareup.test
+
+      import javax.inject.Inject
+
+      abstract class Base1 {
+        @Inject lateinit var string: String
+      }
+
+      abstract class Base2 : Base1() {
+        @Inject lateinit var chars: List<Char>
+      }
+      """
+    ) {
+      assertThat(exitCode).isEqualTo(OK)
+    }
+
+    compile(
+      """
+      package com.squareup.test
+
+      import com.squareup.test.Base1
```

Comment:
This import is not used. I think even `Base2` doesn't need to be imported because the files use the same package.

Reply:
Yep, you're right.  Nice catch!